docs/index.md

```diff
@@ -384,6 +384,13 @@ Spine Default Animation
 Skin
 : The skin to use for the animation when the scene is initialized.
 
+Create Bones
+: When enabled, creates individual GUI nodes for each Spine bone under the Spine node. This allows parenting other GUI nodes to bones and using functions such as [`gui.get_spine_bone()`](/extension-spine/gui_api#gui.get_spine_bone).
+
+- Default for new nodes: Off. Keeps node count low and improves performance.
+- Existing projects: Maintained as On by default for backwards compatibility.
```

Comment:
Hmm, how does that work? I'd assume that to maintain compatibility, you would make the default protobuf value `true`, and fix the template to set it to `false`, but it doesn't look like this is what's happening. How do you do this?